README.md

```diff
@@ -1 +1,76 @@
-# To-Do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo List
+
+> A todo list app made in javascript
+
+![screenshot](thumbnail.png)
+
+Add to-do tasks, mark them as completed and re arrange them in order
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+
+## Getting Started
```

Comment:
- [ ] Great job adding instructions on how to get the local copy of the project up and running. However, I think it could be a good idea to provide some more instructions or commands on how to run the project locally because remember you are using webpack and other dependencies, which means we also need to run the `npm install` along with other commands to run the app. Can you please provide this information in your README? 😉